sw/device/boot_rom/BUILD

```diff
@@ -0,0 +1,73 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU", "opentitan_binary")
+load("//rules:autogen.bzl", "autogen_chip_info")
+
+autogen_chip_info(
+    name = "chip_info",
+)
+
+opentitan_binary(
+    name = "boot_rom",
+    linkopts = [
+        "-T $(location rom_link.ld)",
+    ],
+    output_scrambled = True,
+    deps = [
+        "rom_link.ld",
+        ":boot_rom_lib",
+        "//sw/device/lib/base:mmio",
+    ],
+)
```

Comment:
This doesn't produce a valid executable.  The sources `irq_vector.S` and `rom_crt.S` are omitted, and thus, none of the kept sections in `rom_link.ld` have any references to any other code in the deps and therefore, the linker produces an ELF executable with no code.

When translated to the other forms (bin, dis and scr), these files are essentially empty.

You'll need to add the two assembly files to either `boot_rom` or `boot_rom_lib` and satisfy their dependencies as well (ie: I think rom_crt depends on a bunch of additional hw regs).

You can test that the produced executable is valid by using [verilator](https://docs.opentitan.org/doc/ug/getting_started_verilator/) and substituting in the scrambled ROM image produced by this rule (you can also omit the `--meminit=flash,...` option as we don't need flash just to check that the ROM starts.  You do, however, need the `--meminit=otp,...` option).  Within about 20-30s, you should see the rom output in `uart0.log` (`tail -f` is your friend).